### ISSUE TITLE
Version 2.1.1 is broken - missing configuration in Objects.yaml

### ISSUE THREAD

**Issue by rkrahnen** [Flowpack/jobqueue-common#38]
When updating from `2.1.0` to `2.1.1` an error occurs:
`The object "Neos\Cache\Frontend\VariableFrontend" which was specified as a property in the object configuration of object "Flowpack\JobQueue\Common\Queue\FakeQueue" (automatically registered class) does not exist.`

It's caused by missing the following configuration in `Objects.yaml`:
```
Flowpack\JobQueue\Common\Queue\FakeQueue:
  properties:
    messageCache:
      object:
        factoryObjectName: Neos\Flow\Cache\CacheManager
        factoryMethodName: getCache
        arguments:
          1:
            value: FlowPackJobQueueCommon_MessageCache
```

Could you please provide a new version with this fix?

**Comment by ralfglende** [comment]
+1

**Comment by daniellienert** [comment]
Fixed with #39
